### DISCUSSION TITLE
Display splashscreen while loading data (especially for web export)

### DISCUSSION THREAD

**Discussion by InfiniteProductions** [godotengine/godot-proposals#7149] [General Discussions]
Hi,

when game exported to web is loading (for instance on _itch.io_), there is an ugly empty screen and a lonely progress bar which slowwwwwlllllyyyyyy move toward the end, why not take advantage of loading time, whatever is done behind the scene, to display the **splashscreen** and the progress bar on the bottom (or customizable location and even look) as it was done before when game load on _Amstrad_ or _Amiga_ (actually I'm not sure _CPC_ games dev bother that much about player's patience :), or just in the 90s and early 2000s with PC games) for instance.

It would add a lot for maybe a few (I hope) effort. It could be great to have also more than one splashscreen (for editor, publisher, or whatever third party might be involved in the game).

**Comment by theraot** [comment]
Yes, the splash is what I would want to be the default too.

By the way, it is customizable: https://docs.godotengine.org/en/stable/tutorials/platform/web/customizing_html5_shell.html - and yes I tried getting the splash there once, but could not get it right, and gave up. It was time taken away from actually making the game.
